src/Microsoft.OData.Core/UriParser/Visitors/SelectPropertyVisitor.cs

```diff
@@ -75,7 +75,7 @@ public SelectExpandClause DecoratedExpandClause
         public override void Visit(SystemToken tokenIn)
         {
             ExceptionUtils.CheckArgumentNotNull(tokenIn, "tokenIn");
-            throw new ODataException(ODataErrorStrings.SelectPropertyVisitor_SystemTokenInSelect(tokenIn.Identifier));
+            throw new ODataException(ODataErrorStrings.UriSelectParser_TermIsNotValid(tokenIn.Identifier));

```

Comment:
>UriSelectParser_TermIsNotValid [](start = 55, length = 30)

Is it a breaking change?

Reply:
Actually, I think I updated the exception in the wrong place.

---
In reply to: [166084054](https://github.com/OData/odata.net/pull/1063#discussion_r166084054) [](ancestors = 166084054)